<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QSD88I6" failures="1" tests="5" name="Test" time="66.301" errors="0" timestamp="2023-09-28T15:29:23 IST">
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="test_LoginDDT" time="1.853">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_003_LoginDataDrivenTest.test_LoginDDT(TC_003_LoginDataDrivenTest.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test_LoginDDT -->
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="test_LoginDDT" time="11.263"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="test_LoginDDT" time="11.285"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="test_LoginDDT" time="17.001"/>
  <testcase classname="testCases.TC_003_LoginDataDrivenTest" name="test_LoginDDT" time="15.76"/>
</testsuite> <!-- Test -->
